option_parser/src/lib.rs

```diff
@@ -189,7 +189,7 @@ impl FromStr for IntegerList {
             let items: Vec<&str> = range.split('-').collect();
 
             if items.len() > 2 {
-                return Err(IntegerListParseError::InvalidValue(range.to_string()));
+                return Err(IntegerListParseError::InvalidValue((*range).to_string()));
```

Comment:
Why is this file changed? This has nothing to do with VmmOps, right?

Reply:
Oh, so you attempted to fix clippy warnings in this PR as well.

I think those warnings should be fixed in a separate PR.

Reply:
Okay, these show up when running `dev_cli.sh tests --all`.

The fixes here should really be in a separate PR. They don't need to wait until `VmmOps` is settled.